app/components/markdown/markdown-editor.tsx

```diff
@@ -9,11 +9,12 @@ interface Props {
   label: string;
   name: string;
   disabled: boolean;
+  placeholder?: string;
```

Comment:
@hunar1312 should be required field. Also I see you passed the placeholder to the new item view but you also need to pass it on the edit view.

Reply:
@DonKoko, this branch is not updated with the item edit page.